### PR TITLE
feat: link reliability checks

### PR DESCRIPTION
**Pull request by robertsLando**

### CODE REVIEW

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by AlCalzone** [comment]
this is pending a node-zwave-js release

**Comment by robertsLando** [comment]
Bumped

File: src/components/dialogs/DialogLinkReliability.vue [hunk 1]
```diff
@@ -283,13 +292,10 @@ export default {
 
 			if (response.success) {
 				this.showSnackbar('Link statistics aborted', 'success')
+				this.running = false
 			}
-
-			this.running = false
 		},
 		async checkLinkReliability() {
-			this.running = true
```
Comment (AlCalzone) [inline src/components/dialogs/DialogLinkReliability.vue]: This...

File: src/components/dialogs/DialogLinkReliability.vue [hunk 2]
```diff
@@ -306,10 +312,9 @@ export default {
 				},
 			)
 
-			this.running = false
-
 			if (response.success) {
 				this.statistics = response.result
+				this.running = true
```
Comment (AlCalzone) [inline src/components/dialogs/DialogLinkReliability.vue]: ...and this doesn't make sense. Awaiting `checkLinkReliability` is the entire process, so after it returns, the check is done and is no longer running.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
I swear I won't bother you with lots of polishing for now, but a progress bar (indeterminate state for infinite rounds) would be nice:
grafik

File: src/components/dialogs/DialogLinkReliability.vue (hunk 2 above)
Comment (robertsLando) [reply src/components/dialogs/DialogLinkReliability.vue]: Oh didn't know that I thought the api was only used to start the process!

**Pull request merged**